exchanges/ftx/ftx_wrapper.go

```diff
@@ -682,12 +682,15 @@ func (f *FTX) SubmitOrder(ctx context.Context, s *order.Submit) (order.SubmitRes
 
 // ModifyOrder will allow of changing orderbook placement and limit to
 // market conversion
-func (f *FTX) ModifyOrder(ctx context.Context, action *order.Modify) (*order.Modify, error) {
+func (f *FTX) ModifyOrder(ctx context.Context, action *order.Modify) (*order.ModifyResponse, error) {
 	if err := action.Validate(); err != nil {
 		return nil, err
 	}
 
-	if action.TriggerPrice != 0 {
+	var id string
+	switch {
+	case action.TriggerPrice != 0:
+		var a TriggerOrderData
```

Comment:
I think it would be best to set more details from the response with all of these functions. Including price/amount to account for any issues